tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/LraResource.java

```diff
@@ -96,7 +96,12 @@
     @Produces(MediaType.APPLICATION_JSON)
     @Status
     @LRA(value = LRA.Type.NOT_SUPPORTED)
-    public Response status(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId) {
+    public Response status(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId,
+                           @HeaderParam(LRA_HTTP_RECOVERY_HEADER) URI recoveryId) {
+
+        assertHeaderPresent(lraId, LRA_HTTP_CONTEXT_HEADER); // the TCK expects the coordinator to invoke @Completed methods
```

Comment:
The comment should not mention coordinator and this is Status method but the comment is mentioning Complete. I guess probably a copy-paste error?

Reply:
I just copied the comment from another `assertHeaderPresent` where the term coordinator was already used. 
But indeed, we should no longer use the term.
I'll update (also the copy-paste errors!)
Thx